.github/DISCUSSION_TEMPLATE/help.yml

```
@@ -1,25 +1,36 @@
----
 body:
-  - type: textarea
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to start a new discussion!
+        
+        Before you post:
+        • Check if someone has already asked/answered your question in a previous discussion.
+        
+        When you're ready to post:
+        • Add tags to your discussion (e.g. 'React', 'Vue', 'Vite') to make it clearer for other users.
+        
```

```suggestion
      value: |
Thanks for taking the time to start a new discussion!
        
Before you post:
- Check if someone has already asked/answered your question in a previous discussion.
        
When you're ready to post:
- Add tags to your discussion (e.g. 'React', 'Vue', 'Vite') to make it clearer for other users.
        
```

Why?

This value (as well as `description`s, in other properties) is formatted as markdown. As such, indents of four or more spaces will likely result in that content being wrapped in code block:

_This markdown:_

> ```md
> Text
> 
>     Indented text
>  
>  More text
>  ```

 
 _Produces:_

> Text
>  
>  ```
>  Indented text
>  ```
>  
>  More text

-----

_If you already confirmed that your current content displays as desired, then disregard this comment!_